spec/rails_helper.rb

```diff
@@ -1,5 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'simplecov'
+SimpleCov.start
```

Comment:
we can add `SimpleCov.add_filter ["spec"]` under line 3 to remove running simplecov on test files. simplecov doesn't need to test test files